datasetinsights/datasets/simulation/__init__.py

```diff
@@ -1,6 +1,14 @@
+# flake8: noqa
```

Comment:
Why ignore all falke8 error here?

Reply:
init I have only defined import but not used so flak8 complains.